.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.defaultFormatter":
+}
```

Comment:
Is this settings file something that everyone should have?

If not, let's add `.vscode` to `.gitignore` and remove it from this branch.